tests/v2/integration/setup/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/creasty/defaults"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/token"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+// setup for integration testing
+func main() {
+	rancherConfig := new(rancher.Config)
+	file := "config.yaml"
+
+	user := &management.User{
+		Username: "admin",
+		Password: "admin",
+	}
+
+	hostURL := fmt.Sprintf("%s:8443", getHostIP())
+	token, err := token.GenerateUserToken(user, hostURL)
+	if err != nil {
+		logrus.Fatalf("error with generating admin token: %v", err)
+	}
+
+	cleanup := true
+	rancherConfig.AdminToken = token.Token
+	rancherConfig.Host = hostURL
+	rancherConfig.Cleanup = &cleanup
+
+	if err := defaults.Set(rancherConfig); err != nil {
+		logrus.Fatalf("error with setting up config file: %v", err)
+	}
+
+	all := map[string]*rancher.Config{}
+	all[rancher.ConfigurationFileKey] = rancherConfig
+	yamlConfig, err := yaml.Marshal(all)
+	if err != nil {
+		logrus.Fatalf("error with marshaling config file: %v", err)
+	}
+
+	err = os.WriteFile(file, yamlConfig, 0644)
+	if err != nil {
+		logrus.Fatalf("error with writing config file: %v", err)
+	}
+}
+
+// get host IP address from machine
+func getHostIP() string {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
```

Comment:
Whats the point of specifying the google dns server ? Can host not just be localhost or 127.0.0.1 ?

Reply:
Removed and just put `localhost`